### ISSUE TITLE
A11y_AzureToolsStorageExplorer_ConnectwithNameandKey_Screenreader: NVDA does not read the name for the "Display name edit field"

### ISSUE THREAD

**Issue by Charanrajkb** [microsoft/AzureStorageExplorer#1891]
" [Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing aster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #NVDA; Win32; 

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
Windows10

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect Dialog" pane present on left side of screen and hit enter.
3. Select "Use Storage account name and key" radio button and hit enter on Next button.
4. "Connection with Name and Key" screen gets opened.
5. Start NVDA and navigate using the tab key.
6. Observe the NVDA readings.

​**Actual:​**
NVDA does not read the name for the "Display name edit field
NVDA just reads it as edit and the value present.

**Expected:**
Screen reader must mention the name of the edit field.

**User Impact:**
Screen readers user will not understand the name of the control.

**Recommendation:**
[https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx](url)

**MAS Reference:**
MAS1.3.1:
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

**Attachment for Reference:**
[1891_A11y_AzureToolsStorageExplorer_ConnectwithNameandKey_Screenreader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3646169/1891_A11y_AzureToolsStorageExplorer_ConnectwithNameandKey_Screenreader.pptx)

**Comment by MRayermannMSFT** [comment]
Fix merged into master.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by V-sheman** [comment]
Verified this issue on:Azure Tools Storage Explorer (Version: 1.12.0)
The issue is no longer to reproduces, Hence closing this bug.

**Comment by vinodbk197** [comment]
Verified this issue on:Azure Tools Storage Explorer (Version: 1.12.0)
The issue is no longer to reproduces, Hence closing this bug.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
